packages/svelte/tests/runtime-runes/samples/attribute-spread-call-expression/_config.js

```diff
@@ -8,11 +9,13 @@ export default test({
 		<button class="red">red</button>
 	`,
 
-	async test({ assert, target }) {
+	test({ assert, target }) {
 		const [b1, b2, b3, b4] = target.querySelectorAll('button');
 
-		b1?.click();
-		await Promise.resolve();
+		flushSync(() => {
```

Comment:
Is this the same as 
```js
b1?.click();
flushSync();
```
? Reads a bit better/less to type IMO

Reply:
It took me forever to change them to what it is now lol. I really don't want to have to rewrite them all again.

Reply:
All good 😄